app/javascript/lib/vue-components/modules/BeesWarmChart.vue

```diff
@@ -104,9 +105,14 @@ export default {
   },
   mounted() {
     this.dataWithoutCoordinates = JSON.parse(JSON.stringify(this.data));
-    const containerChart = document.querySelector('.beeswarm-container');
+    const containerChart = document.querySelector('.visualizations-home-main');
```

Comment:
Where is this?

Reply:
[The main class of the Summary](https://github.com/PopulateTools/gobierto/blob/improvent_visualizations/app/javascript/gobierto_visualizations/webapp/containers/contracts/Home.vue#L14)

Reply:
Holy shit. You're breaking all concepts about what a component is!! This is definitely **wrong**.

You're setting a dependency between the component and the container. A component MUST work on its own. Indeed, `BeesWarmChart.vue` is in vue-components library, therefore, if I create a new Beechart, somewhere else.... do I need the `.visualizations-home-main`?? No way.